src/commonMain/kotlin/org/hisp/dhis/rules/models/EnrollmentStatus.kt

```diff
@@ -0,0 +1,12 @@
+package org.hisp.dhis.rules.models
+
+import kotlin.js.ExperimentalJsExport
+import kotlin.js.JsExport
+
+@JsExport
+@OptIn(ExperimentalJsExport::class)
+enum class EnrollmentStatus {
```

Comment:
This name could to collide with `EnrollmentStatus` enum defined in backend and android (at least in Android for sure), which would force to use the full package name when used in the same file. Maybe `RuleEnrollmentStatus`? Or is it unnecessary long?

Reply:
Done